test/e2e/common/node/runtime.go

```diff
@@ -254,28 +258,53 @@ while true; do sleep 1; done
 			})
 		})
 
-		ginkgo.Context("when running a container with a new image", func() {
+		framework.Context("when running a container with a new image", framework.WithSerial(), func() {
```

Comment:
why is it serial? Can we instantiate a new registry for each test?

Reply:
We moved from labeling nodes to running a daemonset, but it still probably needs to be serial to avoid resource conflicts due to the daemonset(s). 

We are assuming that if we run in serial the daemonset + some trivial image pull test pod will fit on any node and therefore do not have to take special care about waht node the test pod runs on.

In practice this probably works without serial, but it's harder to guarantee we won't have resource conflicts etc ..... I think the other daemonset tests are Serial